### DISCUSSION TITLE
How to add languages to Normcap used as an appimage?

### DISCUSSION THREAD

**Discussion by ie377** [dynobo/normcap#467] [Q&A] [answered]
Normcap works great when recognizing English text. How to use it with Russian?

**Comment by dynobo** [comment, marked as answer]
You mean recognizing Russian text in Cyrillic, correct?
To do so click the cogwheel-icon in top right of your screen (it appears during capture), to open the setting menu, then click "Add language" in the language section:

![Screenshot from 2023-06-26 19-48-51](https://github.com/dynobo/normcap/assets/11071876/1b5d9b23-53f7-455c-bcad-f83ebe931010)

Does this help you?

**Reply by ie377** [reply]
wow, very cool! thank you! I did not notice the cogwheel for months and searched in vain for language option in the tray menu))
